Assets/Toast UI/Scripts/Toast.cs

```diff
@@ -110,6 +110,7 @@ public static class Toast {
 
 
       public static void Dismiss () {
+         Prepare () ;
```

Comment:
This will prevent this error:
![image](https://github.com/herbou/Unity_ToastUI/assets/35425887/16f12184-ef3b-477c-b5ec-ceed67cce1d0)